src/Symfony/Component/HttpClient/CachingHttpClient.php

```diff
@@ -86,10 +82,12 @@ public function request(string $method, string $url, array $options = []): Respo
 
             foreach ($values as $cookies) {
                 foreach (explode('; ', $cookies) as $cookie) {
-                    if ('' !== $cookie) {
-                        $cookie = explode('=', $cookie, 2);
-                        $request->cookies->set($cookie[0], $cookie[1] ?? null);
+                    if ('' === $cookie) {
```

Comment:
Why is this better?

Reply:
First reason is to be consistent with the rest of the code of the component.
Second, I always try to move the relevant part of the code out of `if`, to reduce the overall cyclomatic complexity.
Then, on a personal taste, I always find it easier to understand and read this `===` to `!==`.

Reply:
note that the cyclomatic complexity here is *exactly* the same

Reply:
I vote for no change since my personal taste think it is easier to read.

Reply:
Reverted.